run_conformance_tests.py

```diff
@@ -310,9 +316,9 @@ def main(argv=sys.argv[1:]):
     ctest_call = generate_ctest_call(additional_ctest_args)
 
     # Make a build directory if required and enter it
-    if not os.path.isdir('build'):
-        os.mkdir('build')
-    os.chdir('build')
+    if not os.path.isdir(build_dir):
```

Comment:
On a related note, I'm not a huge fan of using `chdir`. I was wondering if we could instead provide cmake and ctest the path to the build directory? They both have flags to do that.

Reply:
Ah so unfortunately it's a little more complicated than I had hoped. We take the `--build-system-call` and execute that unconditionally. I don't think we particularly want to have users pass `-c 'ninja -C <build-dir>' *and* `--build-dir <build-dir>`, do we?

We could still have the `chdir` but avoid the `mkdir`, of course. But at that stage, is it worth it?